app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md

```diff
@@ -0,0 +1,58 @@
+---
+title: No longer asking for a trainee’s home address
+description: We’ve stopped importing, collecting or displaying home addresses.
+date: 2023-05-19
+---
+
+We currently collect a trainee’s home address when:
+
+- we import their data from the Manage teacher training applications service
+- a user manually adds their data to Register
```

Comment:
```suggestion
- a user manually adds their trainees to Register
```

Reply:
The intention here was for "their" in the bullets to refer to the trainee. So it's the trainee's data. However if that isn't clear I'll make a change.